README.md

```diff
@@ -172,7 +172,8 @@ The PPX is included in the `melange-json` package. To use it, just add the
 ```dune
 (library
  (modes melange)
- (preprocess (pps melange-json.ppx)))
+ (preprocess (pps melange-json.ppx))
+ (flags :standard -open Ppx_deriving_json_runtime.Primitives))
```

Comment:
This fixes the problem :) But I think the docs should not encourage global opens using compiler flags, and instead show local opens in the modules where the ppx is used. The reason is that local opens are a more "precise" approach (modules that don't use the PPX won't get these runtime function in scope), plus opening modules using flags like here can lead to really cryptic errors sometimes (if the `pps` field is not set one gets "Unknown location" errors).

Could you please update the readme to include a note on the `Usage` section below that explains it? Including some snippet:

```ocaml
open Ppx_deriving_json_runtime.Primitives
```